projects/go-tester/src/app/components/test-page-3/test-page-3.component.html

```diff
@@ -45,6 +45,18 @@ <h2 class="go-heading-2">
                 ></go-input>
               </div>
 
+              <div class="go-column go-column--100">
+                <go-file-upload
+                  [control]="fileControl"
+                  buttonVariant="positive"
+                  buttonIcon="home"
```

Comment:
It seems like maybe we don't need `buttonVariant` or `buttonIcon` here, correct?